test/integration-test.py

```diff
@@ -156,13 +156,29 @@ def test_ocsp():
     wait_for_ocsp_good(cert_file_pem, "test/test-ca2.pem", ee_ocsp_url)
 
 def test_ct_submission():
-    url = "http://localhost:4500/submissions"
-    submissions = urllib2.urlopen(url).read()
-    expected_submissions = int(submissions)+1
+    url_a = "http://localhost:4500/submissions"
```

Comment:
Please add a comment explaining what this is testing and how it works, especially how it relies on the difference between the CA config and the ocsp-updater config.